cpp/ql/test/library-tests/syntax-zoo/raw_consistency.expected

```diff
@@ -51,7 +50,6 @@ instructionWithoutSuccessor
 | condition_decls.cpp:26:23:26:24 | IndirectMayWriteSideEffect: call to BoxedInt |
 | condition_decls.cpp:41:22:41:23 | IndirectMayWriteSideEffect: call to BoxedInt |
 | condition_decls.cpp:48:52:48:53 | IndirectMayWriteSideEffect: call to BoxedInt |
-| cpp17.cpp:15:5:15:45 | InitializeDynamicAllocation: new |
```

Comment:
These consistency improvements have gone away after the latest refactorings. That's of course a shame, but at least it makes it easier to believe that this PR is very low risk.